commit 833e95351b1f41e739008808f7cc5cff1462a0f7 (refs/changes/95/3895/4)
Author: Rui Loura <rui@joyent.com>
Date:   2018-05-02T19:39:29+00:00 (1 year, 5 months ago)
    
    TRITON-357 sdc-cnapi should log errors encountered when posting tasks to clients
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/models/server.js b/lib/models/server.js
index 376f4c7..76377cc 100644
--- a/lib/models/server.js
+++ b/lib/models/server.js
@@ -2270,7 +2270,11 @@ function (opts) {
         }
     ],
     function (error) {
-        log.info('done posting task to client');
+        if (error) {
+            log.error({err: error}, 'error posting task to client');
+        } else {
+            log.info('done posting task to client');
+        }
         callback(null, taskstatus);
     });
 
